yb-voyager/cmd/metadataDB.go

```diff
@@ -136,7 +142,7 @@ type MetaDB struct {
 }
 
 func NewMetaDB(exportDir string) (*MetaDB, error) {
-	db, err := sql.Open("sqlite3", getMetaDBPath(exportDir))
+	db, err := sql.Open("sqlite3", fmt.Sprintf("%s?_txlock=immediate&_timeout=30000", getMetaDBPath(exportDir)))
```

Comment:
Who duplicates sqlite options like this!!! 🙄 
Define a const `SQLITE_OPTIONS` and then use it to construct the connection string.